sos.spec

```diff
@@ -2,21 +2,23 @@ Summary: A set of tools to gather troubleshooting information from a system
 Name: sos
 Version: 4.5.4
 Release: 1%{?dist}
-Group: Applications/System
-Source0: https://github.com/sosreport/sos/archive/%{name}-%{version}.tar.gz
+Source0: https://github.com/sosreport/sos/archive/%{version}.tar.gz
 License: GPL-2.0-or-later
-BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
 BuildArch: noarch
-Url: https://github.com/sosreport/sos/
+Url: https://github.com/sosreport/sos
 BuildRequires: python3-devel
 BuildRequires: python3-setuptools
-BuildRequires: gettext
 Requires: python3-rpm
+Requires: tar
+Requires: xz
 Requires: python3-pexpect
 Requires: python3-setuptools
 Recommends: python3-magic
+Requires: python3-requests
```

Comment:
`python3-requests` is mandatory just for uploading to a SFTP server. Where Fedora sosreports are uploaded to / what protocol is used (even in Red Hat downstream, we use `Recommends` - do you really need the hard dependency here)?

I am ok with it, just I am asking as I am unsure if the requirement (as is) is such demanded.

Reply:
moving to recommends as well